homeassistant/components/wemo/__init__.py

```diff
@@ -68,11 +70,16 @@ def coerce_host_port(value):
 )
 
 
+@singleton("f{__name__}.registry")
+async def async_get_registry(hass: HomeAssistant) -> pywemo.SubscriptionRegistry:
```

Comment:
Why is this needed? Only a single config entry is supported.

Reply:
Removed.